app/views/companies/show.html.haml

```diff
@@ -47,7 +47,7 @@
     = render 'company_addresses', company: @company, user_can_edit: user_can_edit
 
     .col-md-12
-      - unless @company.dinkurs_company_id.blank?
+      - unless @company.dinkurs_company_id.blank? || !(current_user.member_or_admin? || @company.show_dinkurs_events)
```

Comment:
For an admin or a member of this company, _if_ there is a _dinkurs_company_if_ defined, _and_ `show_dinkurs_events` is false, then the view should not show anything regarding an events table.

That is, instead of seeing this:

---
<img width="742" alt="screen shot 2018-07-01 at 2 18 30 pm" src="https://user-images.githubusercontent.com/9968213/42137417-ab0f2930-7d39-11e8-968a-181d10394be7.png">

---
We should just see this:
```
(Showing Dinkurs events on this page has been disabled. Edit the company to enable that.)
```
(include the parentheses).

This particular statement could be replaced with:
```
if ! @company.dinkurs_company_id.blank? && user_can_edit
```
which is easier to read.  (this will also prevent a member who is _not_ associated with this company from seeing this message).